packages/react-components/styles/fonts.tsx

```diff
@@ -167,6 +167,11 @@ export const fontStyles = StyleSheet.create({
     fontFamily: HindSilguri.Bold,
     color: colors.dark,
   },
+  headerBoldTitle: {
```

Comment:
This is the same as `headerTitle` above it but with one font-size more. Can we merge these two? If not, consider renaming this.

Reply:
Good point- will merge these to one